assignments/arrays.js

```diff
@@ -75,30 +75,91 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+
+  for(i = 0; i < inventory.length; i++){
+    if(inventory[i].id === 33){
+      console.log("Car 33 is a " + inventory[i].car_year + " " + inventory[i].car_make + " " + inventory[i].car_model + ".");
+    }
+  }
+
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
+// function lastCar(inventory){
+//   let lastCar = 0;
+//   for(i = lastCar; i < inventory.length; i++){
+//     if(inventory[i].id === inventory.length)
+//     {
+//       return `${inventory[i].car_make}, ${inventory[i].car_model}`
+//     }
+//   }
+// }
+// console.log(lastCar(inventory));
 let lastCar = 0;
-console.log();
+
+  for (let i = 0; i < inventory.length; i++){
+      if (inventory[i].id === 50) {
```

Comment:
Remember to make this dynamic so if the array changes, it still doesn't just grab the one with id 50.
`inventory[i].id === inventory[inventory.length-1]`